### PR TITLE
[pro] Design polish

### PR DESCRIPTION
**Pull request by wrightmartin**

### CODE REVIEW

**Comment by wrightmartin** [comment]
**Ignore this for now, more polish stuff being added**

**Comment by stevenday** [comment]
There's a few things here that add or change text - just wondering @crowbot whether that will cause issues with the release now?

File: app/views/alaveteli_pro/info_requests/_sidebar.html.erb [hunk 1]
```diff
@@ -17,8 +22,11 @@
                   alaveteli_pro_embargo_path(@info_request.embargo),
                   method: :delete,
                   confirm: "Are you sure you want to publish this request?" %>
```
Comment (wrightmartin) [inline app/views/alaveteli_pro/info_requests/_sidebar.html.erb]: I can't see how to do this - it's different to `<%= _("Embargo") %>` I assume?

File: app/views/alaveteli_pro/info_requests/preview.html.erb [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
             :public_body_name => h(@info_request.public_body.name)) %>
+<div class="pro-canvas">
+  <div class="pro-canvas-header">
+    <div class="row">
+      <h1><%= _('Preview your request') %></h1>
+    </div>
+  </div>
+  <div class="pro-canvas-body">
+    <div class="row">
+      <div class="new_info_request"> <%# TODO - hacky id + class to get same styling as current preview form %>
```
Comment (wrightmartin) [inline app/views/alaveteli_pro/info_requests/preview.html.erb]: Done!

File: app/assets/stylesheets/responsive/alaveteli_pro/_dashboard_layout.scss [hunk 3]
```diff
@@ -24,7 +24,7 @@ $dashboard-collapse: 50em; //where the dashboard layout needs to change
   }
 }
 
-.dashboard-header {
+.pro-canvas-header {
```
Comment (wrightmartin) [inline app/assets/stylesheets/responsive/alaveteli_pro/_dashboard_layout.scss]: This looks out of date for some reason - in my local version they are?

File: app/views/alaveteli_pro/info_requests/_sidebar.html.erb (hunk 1 above)
Comment (wrightmartin) [inline app/views/alaveteli_pro/info_requests/_sidebar.html.erb]: Done!

**Pull request merged**